apiary.geojson

```diff
@@ -0,0 +1,33 @@
+{
+    "type": "FeatureCollection",
+    "features": [{
+        "type": "Feature",
+        "properties": {
+            "id": 1,
+            "name": "Vahekoidu",
+            "address": "Vahekoidu tee 6, Koidu küla, Saue vald, Harju maakond, Estonia",
+            "code": "EE31636",
+            "hives": 3,
+            "hives-type": ["farrar", "langstroth", "other"],
+            "colonies": 3,
+            "colonies-race": ["italian", "carnica", "other"],
+            "status": ["planned", "operating", "abandoned"],
```

Comment:
"planned" could be useful for beekeeper when planning new apiary locations. Is "abandoned" needed (added as a part of apiary's lifecycle)?